client.go

```diff
@@ -62,22 +62,22 @@ func NewClientWithTimeout(timeout time.Duration) *Client {
 type federationTripper struct {
 	// transports maps an TLS server name with an HTTP transport.
 	transports      map[string]http.RoundTripper
-	transportsMutex *sync.Mutex
+	transportsMutex sync.Mutex
 }
 
 func newFederationTripper() *federationTripper {
 	return &federationTripper{
 		transports:      make(map[string]http.RoundTripper),
-		transportsMutex: new(sync.Mutex),
+		transportsMutex: sync.Mutex{},
```

Comment:
isn't this what happens anyway, so this is redundant? or am I confused?

Reply:
Oh yes your're right, sorry.